tests/pytests/integration/states/test_file.py

```diff
@@ -346,3 +415,594 @@ def test_issue_60203(
                 "file_|-credentials exposed via file_|-/tmp/test.tar.gz_|-managed"
             ]["comment"]
         )
+
+
+@pytest.fixture
+def min_patch_ver():
+    return "2.6"
+
+
+def _check_minimum_version(salt_call_cli, minimum_patch_ver):
+    version = salt_call_cli.run("--local", "cmd.run", "patch --version")
+    version = version.json.split()[2]
+    if _LooseVersion(version) < _LooseVersion(minimum_patch_ver):
+        pytest.xfail(
+            "Minimum version of patch not found,"
+            " expecting {}, found {}".format(minimum_patch_ver, version)
+        )
+
+
+@pytest.mark.skip_unless_on_windows
+@pytest.mark.skipif(not salt.utils.path.which("patch"), reason="patch is not installed")
+def test_patch_single_file(salt_call_cli, min_patch_ver, patch_file_dest):
+    """
+    Test file.patch using a patch applied to a single file
+    """
+    _check_minimum_version(salt_call_cli, min_patch_ver)
+    name_file = patch_file_dest / "name_file.txt"
+    source_file = patch_file_dest / "source_file.patch"
+    name_file_contents = """
+    salt
+    patch
+    file
+    """
+    source_file_contents = """
+    1,3c1,5
+    < salt
+    < patch
+    < file
+    ---
+    > salt
+    > will
+    > patch
+    > this
+    > file
+    """
+    sls_contents = """
+    do-patch:
+      file.patch:
+        - name: {name_file}
+        - source: {source_file}
+    """.format(
+        name_file=name_file, source_file=source_file
+    )
+    sls_temp = temp_file("test_patch.sls", sls_contents, patch_file_dest)
```

Comment:
```suggestion
    sls_temp = pytest.helpers.temp_file("test_patch.sls", sls_contents, patch_file_dest)
```
And you can remove the `temp_file` import